docs/pages/guides/_meta.js

```diff
@@ -1,6 +1,6 @@
 export default {
   "replicating-onchain-state": "Replicating onchain state",
   "hello-world": "Hello World",
-  "extending-a-world": "Extending a World",
+  extending: "Extending a World",
```

Comment:
can we keep the `extending-a-world` slug to avoid the need for another redirect and to better capture what this guide is about?